### PR TITLE
fix: hide treemap label if the value is 5 or less

### PR DESCRIPTION
**Pull request by yuda110**
## To Reviewers
- [ ] Skipping reviews is not a problem

## Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

## Description
Hide label text of Treemap if the value is 5 or less

Before / After
스크린샷 2022-03-11 오전 8 23 07 스크린샷 2022-03-11 오전 8 22 24

## Things to Talk About
Because of the ratio calculation, `totalValue` is added as an argument to `drawTreemapChart()`.
```typescript
// before
drawTreemapChart(chart, props.nameOptions, props.valueOptions);

//after
const totalValue = sum(state.filteredData.map(d => d.value));
drawTreemapChart(chart, props.nameOptions, props.valueOptions, totalValue);
```

or, we can get labelBullet from `drawTreemapChart()` and calculate it in `drawChart()`.
```typescript
const totalValue = sum(state.filteredData.map(d => d.value));
const [series, labelBullet] = drawTreemapChart(chart, props.nameOptions, props.valueOptions);
labelBullet.label.adapter.add('text', .....
```

### CODE REVIEW

File: src/data-display/dynamic/dynamic-chart/templates/treemap/index.vue [hunk 1]
```diff
@@ -125,7 +125,8 @@ export default defineComponent<DynamicChartTreemapProps>({
 
             const chart = am4core.create(ctx, TreeMap);
 
-            drawTreemapChart(chart, props.nameOptions, props.valueOptions);
+            const totalValue = sum(state.filteredData.map(d => d.value));
```
Comment (WANZARGEN) [inline src/data-display/dynamic/dynamic-chart/templates/treemap/index.vue:128]: This component doesn't know what key the value is in.
The key path information is in `valueOptions` prop.
Also, must defend against the case where the value is `undefined`.

```suggestion
            const totalValue = sum(state.filteredData.map(d => d[props.valueOptions.key] ?? 0));
```

**Review by WANZARGEN — CHANGES_REQUESTED** [review]
I commented on something that could cause a critical issue. 
Please check it.

**Comment by WANZARGEN** [comment]
I prefer first suggestion.
But the problem is, in updating case, this component only change the data. Not the label bullets.
Please check updating case.

File: src/data-display/dynamic/dynamic-chart/templates/treemap/index.vue (hunk 1 above)
Comment (yuda110) [reply src/data-display/dynamic/dynamic-chart/templates/treemap/index.vue]: Thanks, I changed the code like that.

**Comment by yuda110** [comment]
@WANZARGEN I tested that the label also changes when the data changes.

**Review by WANZARGEN — APPROVED** [review]
Looks good to me :)

**Pull request merged**